snuba/cli/devserver.py

```diff
@@ -144,9 +144,6 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                 [
                     "snuba",
                     "multistorage-consumer",
```

Comment:
let's change this to be just `consumer` (there is only one storage) like in Meredith's PR in #2588

Reply:
no problem, you had the ship it already